services/QuillLMS/engines/evidence/app/services/evidence/gen_ai/repeated_feedback_checker.rb

```diff
@@ -24,7 +24,7 @@ def run
 
       private def system_prompt = Evidence::GenAI::RepeatedFeedbackPromptBuilder.run(prompt: nil, history:)
 
-      private def llm_response = Evidence::OpenAI::Chat.run(system_prompt:, entry: feedback, model: MODEL)
+      private def llm_response = chat_api.run(system_prompt:, entry: feedback, model: chat_api::SMALL_MODEL)
```

Comment:
I really like the idea of using the same interface/expectations for both OpenAI and Gemini Chat services.